### PR TITLE
Apply darkmode style

### LINKED ISSUE #42710: Light/Dark launch background depending on the current theme
## Use case
`Flutter create` automatically creates a pure-white launch background for Android (in `android\app\src\main\res\drawable\launch_background.xml`, with a `@android:color/white` drawable).

## Proposal
Instead, it could use the `?android:attr/colorBackground` theme attribute to have a launch background that depends on the current light or dark theme.

cf https://developer.android.com/guide/topics/ui/look-and-feel/darktheme#launch_screens

### PR DESCRIPTION
**Pull request by younghwan**
## Description
Add darkmode style to the template so that the launch background is applied according to the current theme
|LightMode|DarkMode|
|----------|-----------|
|||

## Related Issues

resolved #42710

## Checklist

Before you create this PR, confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change
Did any tests fail when you ran them? Please read Handling breaking changes.
- [x] No, no existing tests failed, so this is not a breaking change.

### CODE REVIEW

**Comment by younghwan** [comment]
@xster could you check out my PR? Thanks

File: packages/flutter_tools/templates/app/android.tmpl/app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <!-- Theme applied to the Android Window while the process is starting -->
-    <style name="LaunchTheme" parent="@android:style/Theme.Black.NoTitleBar">
+    <style name="LaunchTheme" parent="@android:style/Theme.Light.NoTitleBar">
```
Comment (xster) [inline packages/flutter_tools/templates/app/android.tmpl/app/src/main/res/values/styles.xml:4]: ummm this is reasonable. Thanks for catching this

File: packages/flutter_tools/templates/app/android.tmpl/app/src/main/res/values-night/styles.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Theme applied to the Android Window while the process is starting -->
```
Comment (xster) [inline packages/flutter_tools/templates/app/android.tmpl/app/src/main/res/values-night/styles.xml:3]: Add more context to this comment. e.g. `Theme applied to the Android Window while the process is starting when the OS's Dark Mode setting is on` in this and the next file.

File: packages/flutter_tools/templates/app/android.tmpl/app/src/main/res/values-night/styles.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <!-- Theme applied to the Android Window as soon as the process has started.
+         This theme determines the color of the Android Window while your
+         Flutter UI initializes, as well as behind your Flutter UI while its
+         running.
+         
+         This Theme is only used starting with V2 of Flutter's Android embedding. -->
+    <style name="NormalTheme" parent="@android:style/Theme.Black.NoTitleBar">
+        <item name="android:windowBackground">?android:colorBackground</item>
+    </style>
+</resources>
```
Comment (xster) [inline packages/flutter_tools/templates/app/android.tmpl/app/src/main/res/values-night/styles.xml:18]: Leave the final line break here and in the next file.

**Review by xster — APPROVED** [review]
Thanks for the fix!

File: packages/flutter_tools/templates/app/android.tmpl/app/src/main/res/values/styles.xml (hunk 1 above)
Comment (younghwan) [reply packages/flutter_tools/templates/app/android.tmpl/app/src/main/res/values/styles.xml:4]: Thanks for review! Could you please check again?

**Pull request merged**